### ISSUE TITLE
Incompotable witn pymongo 3.6.0

### ISSUE THREAD

**Issue by tonal** [twisted/txmongo#218]
insert_one error:
```
2017-12-07 12:28:50 [scrapy.utils.signal] ERROR: Error caught on signal handler: <bound method MongodbSaver.spider_opened of <remains.mongo_ext.MongodbSaver object at 0x7f6a10d28208>>
Traceback (most recent call last):
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/home/tonal/lang/projects/remains/remains/mongo_ext.py", line 56, in spider_opened
    ins_ret = yield tasks.insert_one(doc)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/txmongo/collection.py", line 635, in on_ok
    _check_write_command_response([[0, response]])
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/pymongo/helpers.py", line 206, in _check_write_command_response
    write_errors = result.get("writeErrors")
AttributeError: 'list' object has no attribute 'get'
```
replace_one error:
```
2017-12-07 12:29:25 [scrapy.utils.signal] ERROR: Error caught on signal handler: <bound method MongodbSaver.spider_closed of <remains.mongo_ext.MongodbSaver object at 0x7f6a10d28208>>
Traceback (most recent call last):
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/home/tonal/lang/projects/remains/remains/mongo_ext.py", line 84, in spider_closed
    yield tasks.replace_one(doc)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/txmongo/collection.py", line 809, in on_ok
    _check_write_command_response([[0, raw_response]])
  File "/home/tonal/lang/projects/remains/venv3/lib/python3.6/site-packages/pymongo/helpers.py", line 206, in _check_write_command_response
    write_errors = result.get("writeErrors")
AttributeError: 'list' object has no attribute 'get'
```

**Comment by psi29a** [comment]
PyMongo... always breaking things. ;)

**Comment by IlyaSkriblovsky** [comment]
Fixed in #219 

Thanks for report!

**Comment by IlyaSkriblovsky** [comment]
This issue is fixed in TxMongo 18.0.0 that was released today
